packages/docusaurus-theme-search-algolia/package.json

```diff
@@ -12,6 +13,13 @@
     "directory": "packages/docusaurus-theme-search-algolia"
   },
   "license": "MIT",
+  "scripts": {
+    "build": "yarn build:server & yarn build:browser & yarn build:next & yarn build:copy",
+    "build:server": "tsc --project tsconfig.server.json",
+    "build:browser": "tsc --project tsconfig.browser.json",
+    "build:next": "tsc --project tsconfig.browser.json --jsx preserve --outDir lib-next",
+    "build:copy": "node copyUntypedFiles.js"
+  },
```

Comment:
not sure if this is fine, there is no current example of package with build of ts/x files that support both server and browser code

Reply:
next build and browser should be unified, i'm just unsure why *.jsx files are not correctly resolved by webpack

do you have idea?

Reply:
You can output `.js` extension with `"jsx": "react-native"`

Reply:
ok updated